website/docs/r/dns_custom_resolver.html.markdown

```diff
@@ -68,6 +68,7 @@ Review the argument reference that you can specify for your resource.
 - `description` - (Optional, String) Descriptive text of the custom resolver.
 - `high_availability` - (Optional, Bool) High Availability is enabled by Default, Need to add two or more locations.
 - `locations`- (Optional, Set) The list of locations where this custom resolver is deployed. There is no update for location argument in resolver resource.
+- `rule_id` - (String) The rule ID is unique identifier of the custom resolver forwarding rule.
```

Comment:
This rule_id should be in attribute reference inside rules

Reply:
added in Attribute reference